client/components/theme-collection/index.tsx

```diff
@@ -0,0 +1,133 @@
+import { Button } from '@wordpress/components';
+import { Icon, chevronLeft, chevronRight } from '@wordpress/icons';
+import { ReactElement, useEffect, useRef } from 'react';
+import './style.scss';
+import { Swiper as SwiperType } from 'swiper/types';
```

Comment:
There is a [Swiper React](https://swiperjs.com/react) set of components, which I can see in the Calypso node_modules (`wp-calypso/node_modules/swiper/swiper-react.mjs`).

What's the reason for using the vanilla JS version instead?

Reply:
We did use those originally! The swiper people [say they're probably going to remove them](https://swiperjs.com/react) and recommend migrating to the Swiper Element API. The GitHub issues had people struggling with various uses of the Swiper Element API so I stuck with ye olde vanilla js.

Reply:
I guess not a biggie, although I liked the React-y version.

Their warning is super vague ("likely", "in the future", etc.), and I wonder whether we would be safe with it for the time being. 🤔 

Oh well, if this works, then never mind my personal preferences. 🙂

Reply:
I liked it too, and I may just have the wrong instincts given our approach to dependency updates etc.

Using the react version seems like storing up a problem for the future. Swiper seems to move faster than we do in terms of only supporting the latest stuff and dropping support for older stuff. The last thing we want is more work when we're reacting to a security problem in 2028 when we're jumping from 10.1 to 20.1